packages/amplify_authenticator/lib/src/utils/country_code.dart

```diff
@@ -13,231 +13,234 @@
-  Country('PE', '51', 'Peru (+51)'),
-  Country('PH', '63', 'Philippines (+63)'),
-  Country('PL', '48', 'Poland (+48)'),
-  Country('PT', '351', 'Portugal (+351)'),
-  Country('PR', '1787', 'Puerto Rico (+1787)'),
-  Country('QA', '974', 'Qatar (+974)'),
-  Country('RE', '262', 'Reunion (+262)'),
-  Country('RO', '40', 'Romania (+40)'),
-  Country('RU', '7', 'Russia (+7)'),
-  Country('RW', '250', 'Rwanda (+250)'),
-  Country('SM', '378', 'San Marino (+378)'),
-  Country('ST', '239', 'Sao Tome & Principe (+239)'),
-  Country('SA', '966', 'Saudi Arabia (+966)'),
-  Country('SN', '221', 'Senegal (+221)'),
-  Country('CS', '381', 'Serbia (+381)'),
-  Country('SC', '248', 'Seychelles (+248)'),
-  Country('SL', '232', 'Sierra Leone (+232)'),
-  Country('SG', '65', 'Singapore (+65)'),
-  Country('SX', '1', 'Sint Maarten (+1)'),
-  Country('SK', '421', 'Slovak Republic (+421)'),
-  Country('SI', '386', 'Slovenia (+386)'),
-  Country('SB', '677', 'Solomon Islands (+677)'),
-  Country('SO', '252', 'Somalia (+252)'),
-  Country('ZA', '27', 'South Africa (+27)'),
-  Country('ES', '34', 'Spain (+34)'),
-  Country('LK', '94', 'Sri Lanka (+94)'),
-  Country('SH', '290', 'St. Helena (+290)'),
-  Country('KN', '1869', 'St. Kitts (+1869)'),
-  Country('SC', '1758', 'St. Lucia (+1758)'),
-  Country('SD', '249', 'Sudan (+249)'),
-  Country('SR', '597', 'Suriname (+597)'),
-  Country('SZ', '268', 'Swaziland (+268)'),
-  Country('SE', '46', 'Sweden (+46)'),
-  Country('CH', '41', 'Switzerland (+41)'),
-  Country('SI', '963', 'Syria (+963)'),
-  Country('TW', '886', 'Taiwan (+886)'),
-  Country('TJ', '7', 'Tajikstan (+7)'),
-  Country('TH', '66', 'Thailand (+66)'),
-  Country('TG', '228', 'Togo (+228)'),
-  Country('TO', '676', 'Tonga (+676)'),
-  Country('TT', '1868', 'Trinidad & Tobago (+1868)'),
-  Country('TN', '216', 'Tunisia (+216)'),
-  Country('TR', '90', 'Turkey (+90)'),
-  Country('TM', '7', 'Turkmenistan (+7)'),
-  Country('TM', '993', 'Turkmenistan (+993)'),
-  Country('TC', '1649', 'Turks & Caicos Islands (+1649)'),
-  Country('TV', '688', 'Tuvalu (+688)'),
-  Country('UG', '256', 'Uganda (+256)'),
-  Country('UA', '380', 'Ukraine (+380)'),
-  Country('AE', '971', 'United Arab Emirates (+971)'),
-  Country('UY', '598', 'Uruguay (+598)'),
-  Country('UZ', '7', 'Uzbekistan (+7)'),
-  Country('VU', '678', 'Vanuatu (+678)'),
-  Country('VA', '379', 'Vatican City (+379)'),
-  Country('VE', '58', 'Venezuela (+58)'),
-  Country('VN', '84', 'Vietnam (+84)'),
-  Country('VG', '84', 'Virgin Islands - British (+1284)'),
-  Country('VI', '84', 'Virgin Islands - US (+1340)'),
-  Country('WF', '681', 'Wallis and Futuna (+681)'),
-  Country('YE', '969', 'Yemen (North) (+969)'),
-  Country('YE', '967', 'Yemen (South) (+967)'),
-  Country('ZM', '260', 'Zambia (+260)'),
-  Country('ZW', '263', 'Zimbabwe (+263)')
+  Country('US', '1', CountryResolverKey.unitedStates),
+  Country('GB', '44', CountryResolverKey.unitedKingdom)
+  // Country('DZ', '213', 'Algeria (+213)'),
```

Comment:
I assume the rest of these will be added back before the PR is complete?

Reply:
I was going to do this as a separate PR, since it will be a ton of repetition.